### PR TITLE
+act #20936 add CompletionStage API to CircuitBreaker

### PR DESCRIPTION
**Pull request by ktoso**
Resolves https://github.com/akka/akka/issues/20936

### CODE REVIEW

File: akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala [hunk 1]
```diff
@@ -219,7 +219,7 @@ class CircuitBreakerSpec extends AkkaSpec with BeforeAndAfter {
       val breaker = CircuitBreakerSpec.shortCallTimeoutCb()
 
       val fut = breaker().withCircuitBreaker(Future {
-        Thread.sleep(150.millis.dilated.toMillis);
+        Thread.sleep(150.millis.dilated.toMillis)
```
Comment (johanandren) [inline akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala]: much boyscout, wow

**Comment by johanandren** [comment]
LGTM

File: akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala (hunk 1 above)
Comment (ktoso) [inline akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala]: one down, `1278` to go

**Comment by ktoso** [comment]
PLS BUILD

**Comment by ktoso** [comment]
PLS BUILD

**Pull request merged**